papis/commands/__init__.py

```diff
@@ -86,7 +86,7 @@ def get_command(self,
 class Script(NamedTuple):
     """A papis command plugin or script.
 
-    These plugins are made available through the main papis command-line as
```

Comment:
I think this might be a case where "`papis`" is more appropriate as it seems to be talking about the command (maybe it should also be 'command' instead of 'command-line'?).

Reply:
I thought about that exact case. I didn't know if it was referring to `papis` as the command. So formatting and removing will make it clearer.

Reply:
Yeah, I agree it's not a clear case. Best to make it clear :grinning: